python/cudf/cudf/core/series.py

```diff
@@ -1621,7 +1608,11 @@ def _concat(cls, objs, axis=0, index=True):
             if isinstance(objs[0].index, cudf.MultiIndex):
                 index = cudf.MultiIndex._concat([o.index for o in objs])
             else:
-                index = cudf.core.index.Index._concat([o.index for o in objs])
+                with warnings.catch_warnings():
+                    warnings.simplefilter("ignore", FutureWarning)
```

Comment:
What FutureWarning is being caught here?

Reply:
This is filtering the warning for appending empty index objects.